config.md

```diff
@@ -61,7 +61,7 @@ This specification uses the following terms:
 Here is an example image JSON file:
```

Comment:
On Wed, Sep 07, 2016 at 01:22:18PM -0700, Vincent Batts wrote:
> Otherwise I can drop the commit and we can continue with the
> overlapping PRs.

@philips, can we just drop the renaming portion of your #231 (and spin
that out into a separate PR)?